plugins/services/src/js/service-configuration/PodContainerConfigSection.js

```diff
@@ -65,7 +65,7 @@ const PodContainerConfigSection = ({containerConfig, appConfig, onEditClick, ind
       </ConfigurationMapRow>
 
       {/* Resources */}
-      {fields.resources.cpus && (
+      {!!fields.resources.cpus && (
```

Comment:
Ok, I would prefer us casting it to booleans with `Boolean()`, but in general, I am still not sold on this pattern.
It was not introduced here, so it should not hold this PR up, but I feel like this was just introduced before we had a discussion about it.